worlds/oot_soh/location_access/overworld/kokiri_forest.py

```diff
@@ -142,8 +143,11 @@ def set_region_rules(world: "SohWorld") -> None:
     ## KF Outside Deku Tree
     # Locations
     add_events(Regions.KF_OUTSIDE_DEKU_TREE, world, [
-        (EventLocations.MIDO_OUTSIDE, LocalEvents.MIDO_SWORD_AND_SHIELD, lambda bundle: has_item(Items.KOKIRI_SWORD, bundle) and
-                                                  has_item(Items.DEKU_SHIELD, bundle)),
+        (EventLocations.MIDO_OUTSIDE, Events.CAN_FARM_NUTS, lambda bundle: (can_get_deku_baba_nuts(bundle))),
+        (EventLocations.MIDO_OUTSIDE, Events.CAN_FARM_STICKS, lambda bundle: (can_get_deku_baba_sticks(bundle))),
+        (EventLocations.MIDO_OUTSIDE, LocalEvents.MIDO_SWORD_AND_SHIELD, lambda bundle: (has_item(Items.KOKIRI_SWORD, bundle) and 
+                                                                                         has_item(Items.DEKU_SHIELD, bundle)))
+        (EventLocations.MIDO_OUTSIDE, Events.CAN_ACCESS_FAIRIES, lambda bundle: (call_gossip_fairy_except_suns(bundle))),
```

Comment:
EventLocations need to be unique, and also these all wouldn't have been MIDO_OUTSIDE regardless :P